test/benchee/formatters/console_test.exs

```diff
@@ -56,6 +65,73 @@ defmodule Benchee.Formatters.ConsoleTest do
   end
 
   describe ".format_scenarios" do
+    test "displays extended options" do
+      scenarios = [
+        %Scenario{
+          job_name: "First job",
+          run_time_statistics: %Statistics{
+            average: 200.0,
+            ips: 5_000.0,
+            std_dev_ratio: 0.1,
+            median: 195.5,
+            percentiles: %{99 => 300.1},
+            minimum: 111.1,
+            maximum: 333.3,
+            mode: 201.2,
+            sample_size: 50_000
+          }
+        }
+      ]
+
+      [_header1, _result1, descriptor1, descriptor2, result2] =
+        Console.format_scenarios(scenarios, @console_config_extended_params)
+
+        assert descriptor1 =~ ~r/Extended options: /
+        assert descriptor2 =~ ~r/minimum/
+        assert descriptor2 =~ ~r/maximum/
+        assert descriptor2 =~ ~r/sample size/
+        assert descriptor2 =~ ~r/mode/
+        assert result2 =~ ~r/First job/
+        assert result2 =~ ~r/111.10/
+        assert result2 =~ ~r/333.30/
+        assert result2 =~ ~r/50000/
+        assert result2 =~ ~r/201.20/
+    end
+
+    test "displays extended options with multiple mode ouput" do
+      scenarios = [
+        %Scenario{
+          job_name: "First job",
+          run_time_statistics: %Statistics{
+            average: 200.0,
+            ips: 5_000.0,
+            std_dev_ratio: 0.1,
+            median: 195.5,
+            percentiles: %{99 => 300.1},
+            minimum: 111.1,
+            maximum: 333.3,
+            mode: [201.2, 205.55],
+            sample_size: 50_000
+          }
+        }
+      ]
+
+      [_header1, _result1, descriptor1, descriptor2, result2] =
+        Console.format_scenarios(scenarios, @console_config_extended_params)
+
+        assert descriptor1 =~ ~r/Extended options: /
+        assert descriptor2 =~ ~r/minimum/
+        assert descriptor2 =~ ~r/maximum/
+        assert descriptor2 =~ ~r/sample size/
+        assert descriptor2 =~ ~r/mode/
+        assert result2 =~ ~r/First job/
+        assert result2 =~ ~r/111.10/
+        assert result2 =~ ~r/333.30/
+        assert result2 =~ ~r/50000/
+        assert result2 =~ ~r/201.20/
+        assert result2 =~ ~r/205.55/
```

Comment:
I think we can omit the repeated asserts from the test before here and for the mode output actually test it as one regex `201.20 units, 205.55` or something like that :)

Reply:
Yeah, good call, otherwise it obfuscates the actual behavior under test.